deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
```

Comment:
Yes, that's true, but I thought it will be changed in any case. Maybe putting x.x.x.x instead?